enviro/boards/indoor.py

```diff
@@ -44,8 +46,17 @@ def get_sensor_readings(seconds_since_last):
   data = bme688.read()
 
   temperature = round(data[0], 2)
-  pressure = round(data[1] / 100.0, 2)
   humidity = round(data[2], 2)
+
+  # Compensate for additional heating when on usb power - this also changes the
+  # relative humidity value.
+  if config.usb_power:
+    adjusted_temperature = temperature - config.usb_power_temperature_offset
```

Comment:
This should probably be checked before being used, as it would be absent from an old config file.

Reply:
Indeed. See my comment to the main PR